Rakefile

```diff
@@ -1,16 +1,11 @@
 require 'bundler/setup'
 require 'bump/tasks'
 require 'rake/testtask'
-require 'bundler/gem_tasks'
```

Comment:
Why this removal?

Reply:
Good spot. Appraisal was originally being called from the default rake task and reloading bundler causing issues. It was since fixed but I neglected to put this back.

Reply:
Ah, turns out it's still a problem. See 188074917dda45e62361e4c4138867270a466609